### ISSUE TITLE
No bounding box showing while testing on custom data set 

### ISSUE THREAD

**Issue by mansi-aggarwal-2504** [fizyr/keras-retinanet#1479]
Here are the steps I followed:
- downloaded the pretrained model `resnet50_coco_best_v2.1.0.h5`
- then I trained it on my dataset using `keras_retinanet/bin/train.py csv /path/to/csv/file/containing/annotations /path/to/csv/file/containing/classes
` 
- then I converted the 50th epoch model saved in snapshots for inference using `keras_retinanet/bin/convert_model.py snapshots/resnet50_csv_50.h5 /path/to/save/inference/model.h5`
- I used `examples/resnet50_retinanet.py` to test `/path/to/save/inference/model.h5` on some images

The thing is that I am not getting any bounding boxes on any image. Scores are an array of -1s and the boxes also look wrong. I changes `labels_to_names` in the script to my label (there's just one label in my images). I tried to follow steps from the readme and also resolved some errors using FAQ but am I missing something here. Did I have to mention something explicitly while training the `resnet50_coco_best_v2.1.0.h5` on my dataset?
`

**Comment by parthkvv** [comment]
Facing similar issue, were you able to fix the issue @mansi-aggarwal-2504 ?
